sqlalchemy_views/views.py

```diff
@@ -32,7 +32,13 @@ class CreateView(_CreateDropBase):
 
     def __init__(self, element, selectable, on=None, bind=None,
                  or_replace=False, options=None):
-        super(CreateView, self).__init__(element, on=on, bind=bind)
+        try:
+            super(CreateView, self).__init__(element, on=on, bind=bind)
+        except TypeError:
+            # Since version 1.4.0 of SQLAlchemy the ** on ** parameter no
+            # longer exists. it causes a ** TypeError ** exception
+            super(CreateView, self).__init__(element, bind=bind)
```

Comment:
We should probably add a test here for `if on is not None` and raise an exception specifying that `on` is not supported on SQLAlchemy 1.4+. I'm concerned about silently ignoring `on` once folks upgrade to SQA 1.4.